packages/models-library/src/models_library/projects.py

```diff
@@ -149,6 +158,15 @@ class Project(ProjectCommons):
     # Dev only
     dev: Optional[Dict] = Field(description="object used for development purposes only")
 
+    boot_options: Optional[Dict[NodeIDStr, Dict[EnvVarKey, str]]] = Field(
```

Comment:
why not adding this directly on the node instead of the project?

Reply:
That would have been an option, but by design with @mguidon and @odeimaiz we agreed on this.